### DISCUSSION TITLE
How to register a global property through plugin config

### DISCUSSION THREAD

**Discussion by coreybruyere** [DavidWells/analytics#365] [Q&A]
I'm using the `analytics` plugin with mixpanel. Is there a way to pass a custom property to mixpanel from the initialized plugin ? Example: Passing a property of `site` and value of `blog` in one place within plugin instantiation so that all events (page, track, identity) pass that custom property to mixpanel?

```
export const analytics = Analytics({
    app: 'website',
    debug: process.env.NODE_ENV !== 'production',
    plugins: IS_BROWSER ? [
        mixpanelPlugin({
            token: TOKEN,
        }),
    ] : []
});
```

**Comment by DavidWells** [comment]
You can enrich any payload to a given provider https://getanalytics.io/tutorials/enriching-data/
